adapters/google_gemma_2/9b_it/dataloop.json

```diff
@@ -0,0 +1,100 @@
+{
+  "name": "google-gemma2-27b",
+  "displayName": "Google Gemma2 27b",
+  "version": "0.1.22",
+  "scope": "public",
+  "description": "Google Gemma2-27b from NIM API",
```

Comment:
OK. So the json for "Google Gemma2-27b" was replaced with "Google Gemma2-9b from NIM API"